plugins/plugins.go

```diff
@@ -25,8 +32,18 @@ type Plugin interface {
 type Manager struct {
 	Labels   map[string]string
 	Store    storage.Store
+	Compiler *ast.Compiler
```

Comment:
I think the compiler should not be exported since you need to lock the Manager to grab it.

Reply:
un-exported compiler